client/app/auth/auth.js

```diff
@@ -9,7 +9,7 @@ angular.module('boundless.auth', [])
 		Auth.signin($scope.user)
 			.then(function(token) {
 					//this is where we will will set a token/cookie/jwt
-				// $window.localStorage.setItem('our url here', token);
+				$window.localStorage.setItem('our url here', token);
```

Comment:
I notice you're using "our url here", but we can actually use any key. The browser already controls by domain, so we can only see the items we set.